anda/devs/zed-nightly/update.rhai

```diff
@@ -0,0 +1,9 @@
+if filters.contains("nightly") {
+  rpm.global("commit", gh_commit("zed-industries/zed"));
+  if rpm.changed() {
+    let v = find("(\\d+\\.\\d+\\d+\\.\\d+)", find("\nversion = \"(\\d+\\.\\d+\\d+\\.\\d+)\"\n", gh_rawfile("zed-industries/zed", "main", "crates/zed/Cargo.toml"), 1), 1);
```

Comment:
Is this double nested find() collapsible?

Reply:
I don't think so.

Reply:
the outer find() doesn't actually do anything I think?